src/main/java/com/arpnetworking/tsdcore/sinks/HttpSinkActor.java

```diff
@@ -164,6 +169,17 @@ private void processCompletedRequest(final PostComplete complete) {
         _inflightRequestsCount--;
         final Response response = complete.getResponse();
         final int responseStatusCode = response.getStatusCode();
+
+        try (Metrics metrics = _metricsFactory.create()) {
+            final int responseStatusClass = responseStatusCode / 100;
+            metrics.incrementCounter(_requestSuccessName, responseStatusClass == 2 ? 1 : 0);
```

Comment:
I believe success is defined below by `ACCEPTED_STATUS_CODES`. Also, see my note below about splitting the measurement of success across two methods and my original comment about needing to do some refactoring to support this metric.